mgmt/etc/crds/ftns.crd.yaml

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020 The Authors.
+
+# Authors: Cathy Lu <@clu2>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:The above copyright
+# notice and this permission notice shall be included in all copies or
+# substantial portions of the Software.THE SOFTWARE IS PROVIDED "AS IS",
+# WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+# TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+# FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+# TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR
+# THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: ftns.mizar.com
+spec:
+  scope: Namespaced
+  group: mizar.com
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  names:
+    kind: Ftn
+    plural: ftns
+    singular: ftn
+    shortNames:
+      - ftn
+      - ftns
+  validation:
+    openAPIV3Schema:
+      properties:
+        name:
+          type: string
+        ip:
+          type: string
+        droplet:
+          type: string
+        mac:
+          type: string
+        chains:
+          type: array
+          items:
+            type: string
+          minItems: 1
+        nextnode:
+          type: string
+  additionalPrinterColumns:
+    - name: ip
+      type: string
+      priority: 0
+      JSONPath: .spec.ip
+      description: The IP address of ftn chain node
+    - name: droplet
+      type: string
+      priority: 0
+      JSONPath: .spec.droplet
+      description: The droplet where this ftn chain node deployed
+    - name: mac
+      type: string
+      priority: 0
+      JSONPath: .spec.mac
+      description: The Mac address of this ftn chain node
+    - name: chains
+      type: string
+      priority: 0
+      JSONPath: .spec.chains
+      description: The chain(s) that has this ftn chain node
```

Comment:
How does this look like with ```kubectl get```. Since we are now using the OpenAPI v3 validation, let's select minimal data to show here.